js/src/util/subscribe-to-events.spec.js

```diff
@@ -95,21 +95,20 @@ describe('util/subscribe-to-events', () => {
     expect(api.eth.uninstallFilter.firstCall.args).to.eql([ 123 ]);
   });
 
-  it.skip('checks for new events regularly', async function () {
+  it('checks for new events regularly', async function () {
     const { api, contract } = this;
     api.eth.getFilterLogs = stub().resolves([]);
 
     const onLog = spy();
     const onBar = spy();
-    const s = subscribeToEvents(contract, [ 'Bar' ], { interval: 5 })
+    subscribeToEvents(contract, [ 'Bar' ], { interval: 5 })
       .on('log', onLog)
       .on('Bar', onBar);
-    await delay(9);
-    s.unsubscribe();
+    await delay(10);
```

Comment:
> Need to be sure promise are resolved or event emitted […]

The promises definitely resolve on next tick, because they are mocked.

> […] currently everything just waits for some hopefully long enough time instead.

That the events fire within a certain time period is actually what this test tries to assert.